test/deployer/deployer.js

```diff
@@ -111,7 +111,7 @@ describe('Deployer tests', () => {
 			})
 
 			it('should deploy contract with params correctly', async () => {
-				const contractWrapper = await deployer.deploy(VestingContract, config.randomAddress, 1569426974);
+				const contractWrapper = await deployer.deploy(VestingContract, {}, config.randomAddress, 1569426974);
```

Comment:
This is and line 157 are the only two places where you need this `{}`, you should also be able to pass `null`, `false`, or even `undefined` or in general any Falsy value as more meaningful indicator of no linking. Everywhere else it should skip this param